packages/lib/CalEventParser.ts

```diff
@@ -176,38 +177,40 @@ export const getRescheduleLink = (calEvent: CalendarEvent): string => {
   return `${WEBAPP_URL}/reschedule/${seatUid ? seatUid : Uid}`;
 };
 
-export const getRichDescription = (calEvent: CalendarEvent /*, attendee?: Person*/) => {
+export const getRichDescription = (calEvent: CalendarEvent, t_?: TFunction /*, attendee?: Person*/) => {
+  const t = t_ ?? calEvent.organizer.language.translate;
```

Comment:
I would recommend making t_ required here so that from everywhere we are passing the language explicitly and it would not use wrong language.